standard/statements.md

```diff
@@ -604,6 +605,19 @@ Local function bodies are always reachable. The endpoint of a local function dec
 
 If the type of the argument to a local function is `dynamic`, the function to be called must be resolved at compile time, not runtime.
 
+A static local function
+
+- May not reference instance members from an implicit or explicit `this` or `base` reference.
+- May reference `static` members from the enclosing scope.
+- May reference `constant` definitions from the enclosing scope.
+- May use `nameof()` to reference locals, parameters, or `this` or `base` from the enclosing scope.
+- May capture state from an enclosing static local function, but may not capture state outside the enclosing static local function. The same rules apply to a lambda.
+- May not be invoked in an expression tree.
```

Comment:
```suggestion
- Shall not be invoked in an expression tree.
```

Reply:
Is it valid to invoke a non-static local function from an expressoin tree?

Reply:
No, it's not. Local functions (static or not) aren't allowed in expression trees. We missed this in 7.3